gulpfile.js

```diff
@@ -68,47 +70,64 @@ gulp.task('watch', function () {
 
 gulp.task('default', ['css', 'copy', 'lanyrd', 'generate']);
 
+fetchContentFromUrl = async (url) => {
+    const res = await fetch(url)
+    const body = await res.text()
+    return body
+}
+
 var talks = require('./src/content/talks.json');
 var authors = require('./src/content/authors.json');
 var jobs = require('./src/content/jobs.json');
-
-var pages = [
-    {
-        title: 'Home',
-        fileBasename: 'index.ejs',
-        description: "The innovation of a startup combined with the authority of our journalism. The developers of the Guardian are shaping the future of news",
-        jobs: jobs
-    },
-    // {
-    //     title: 'Engineering',
-    //     fileBasename: 'engineering.ejs'
-    // },
-    {
-        title: 'Diversity & Inclusion',
-        fileBasename: 'diversity-&-inclusion.ejs'
-    },
-    {
-        title: 'Open People',
-        fileBasename: 'open-people.ejs'
-    },
-    {
-        title: 'Open Source',
-        fileBasename: 'open-source.ejs'
-    },
-    {
-        title: 'Events & Talks',
-        fileBasename: 'events-&-talks.ejs',
-        talks: talks
-    }
-];
+var engineeringCultureHtml = Marked(fs.readFileSync('./src/content/engineering-culture.md', { encoding: 'utf8' }));
+
+var pages = {
+    navPages: [
+        {
+            title: 'Home',
+            fileBasename: 'index.ejs',
+            description: "The innovation of a startup combined with the authority of our journalism. The developers of the Guardian are shaping the future of news",
+            jobs: jobs
+        },
+        // {
+        //     title: 'Engineering',
+        //     fileBasename: 'engineering.ejs'
+        // },
+        {
+            title: 'Diversity & Inclusion',
+            fileBasename: 'diversity-&-inclusion.ejs'
+        },
+        {
+            title: 'Open People',
+            fileBasename: 'open-people.ejs'
+        },
+        {
+            title: 'Open Source',
+            fileBasename: 'open-source.ejs'
+        },
+        {
+            title: 'Events & Talks',
+            fileBasename: 'events-&-talks.ejs',
+            talks: talks
+        },
+    ],
+    contentPages: [
+        {
+            title: 'Engineering Culture',
+            fileBasename: 'engineering-culture.ejs',
```

Comment:
`engineering-culture.ejs` looks like it is going to be a more generic template for certain types of files, is there a way to denote it as such?